origins_workflow/composer.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "dof-dss/origins_workflow",
+  "description": "Drupal module to provide workflow functionality for NICS websites",
+  "type": "drupal-module",
+  "authors": [
+    {
+      "name": "Martin Dutton",
+      "email": "mail@martinduttonconsulting.co.uk"
```

Comment:
Not a point to hold this up on, but perhaps we should use a generic NICS address for this?

Reply:
I think in future we should just rip out the addresses altogether

Reply:
Nice. I'll create a new ticket to cover that across all repos